pkg/fakes/config/kubeconfig1.yaml

```diff
@@ -12,6 +12,10 @@ clusters:
       insecure-skip-tls-verify: true
       server: https://foo.org:443
     name: foo-cluster
+  - cluster:
+      insecure-skip-tls-verify: true
```

Comment:
nit: would removing this line impact testing the functionality? If not I'd take it out.

Reply:
It shouldn't affect the the testing functionality. Removed it. Thanks